examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -49,6 +49,9 @@ const BACKEND_MODULE_NAME: &str = "backend_module";
 const BACKEND_ENTRYPOINT_NAME: &str = "backend_oak_main";
 const STORAGE_NAME_PREFIX: &str = "abitest";
 
+const HTTP_ADDR: &str = "[::]:8383";
+const ADDITIONAL_TEST_SERVER_ADDR: &str = "[::]:8081";
```

Comment:
I'm confused as to why these have such different names.

- is only the first one an HTTP adress?
- is only the second one for test?
- is only the second one for a server?
- additional with respect to what?
- why do we need two distinc one in the first place?

Reply:
The abitest starts a gRPC server node, and as of this PR an HTTP server node. These are used to examine the main (happy path) functionality via the CPP client. In addition, some tests temporarily create additional gRPC and HTTP server nodes, for testing error checking scenarios. These additional servers need to be started on a different port. 

This is what I understand from the code. It would have been helpful to have the overall scenario documented at a high-level.